src/py42/clients/alerts.py

```diff
@@ -42,23 +45,42 @@ def get_details(self, alert_ids):
     def resolve(self, alert_ids, reason=None):
         """Resolves the alerts with the given IDs.
 
+        Deprecated. Support will be removed from version 2.0.0. Use :meth:`~py42.clients.alerts.AlertClient.update_state()` instead.
```

Comment:
I think whether or not we deprecate any of these methods is a decision for another PR that we should complete before we start work on 2.0